### PR TITLE
Readds the shiver emote

### LINKED ISSUE #22903: Shiver emote not working
HELP: whatever is effecting me keeps trying to use *shiver, which is apparently not a thing 

@lzimann

### PR DESCRIPTION
**Pull request by lzimann**
Fixes #22903

I probably forgot about it.

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -253,6 +253,12 @@
 	message = "shakes their head."
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/shiver
+	key = "shiver"
+	key_third_person = "shiver"
+	message = "shivers."
+	emote_type = EMOTE_AUDIBLE
```
Comment (AnturK) [inline code/modules/mob/living/emote.dm]: You can hear someone shiver ?

Comment (RemieRichards) [inline code/modules/mob/living/emote.dm]: probably the "brrrr" noise people sometimes make when they shiver, at a guess.
that or it's purely for mechanical reasons (sight/deaf checks).

Comment (lzimann) [inline code/modules/mob/living/emote.dm]: it was marked as audible before and I just copied

**Comment by Supermichael777** [comment]
being cold does not make you shiver/10

**Pull request merged**